_posts/2022-12-02-101-robot-design.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Robots 101 - Robot Design
+---
+
+Welcome back to our Robots 101 series! This time we will be looking at what you need to consider when designing a robot that is both practical for the game’s task and one that would fit our safety regulations.
+
+The aim of this post is to give you a jumping off point rather than prescribe the best way to tackle Student Robotics. There will be things we've missed and may even be things that are not relevant to your specific team. Over the years teams have tackled the competition in all sorts of ways we never imagined. Please do experiment and come up with the best way for your team to work.
+
+## Strategy
+
+Before settling on a final design, it’s a good idea to devise several different strategies. Making a scale plan or model of the arena, tokens, and robots is a very good way to start thinking about these and can aid discussion. Be sure to document the pros and cons of each strategy. Include point scoring, ease of implementation, additional items required to implement the strategy, and possible interactions with other robots.
+
+Narrow the list and prototype some hardware (can be simple) to assess feasibility. The most complex, technologically advanced strategy is not necessarily the best. A simple (and elegant) strategy is more straightforward, quicker to implement and less likely to fail. On many occasions, a simple strategy has won the day.
+
+## Movement
+
+How your robot will move is a key decision to make. Many options are available, and all have pros and cons: wheels, tracks, or legs, 2, 3 or 4-wheel drive, omniwheels etc. Types of motor: DC, geared, stepper motors, built-in or add-on encoders. You should research these methods and decide what will be best for your strategy. The ability to move a defined distance and/or rotate a specified number of degrees can be helpful.
+
+## Construction
+
+The construction of your robot will be strongly influenced by the movement system you have decided upon. When planning the construction of your robot, a few things would be good to think about: the strength of your robot, how heavy it will be, the ease of modification or repairs, do you have sufficient ground clearance, is the robot protected from impacts etc.
```

Comment:
```suggestion
The construction of your robot will be strongly influenced by the movement system you have decided upon. When planning the construction of your robot, a few things would be good to think about: the strength of your robot, how heavy it will be, the ease of modification or repairs, do you have sufficient ground clearance, is the robot protected from impacts, etc.
```